### PR TITLE
BROKER: Insert Cars

### PR DESCRIPTION
**Pull request by MananSThakkar**

### CODE REVIEW

File: CarApp.ConsoleApp/Models/Cars/Cars.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace Car.ConsoleApp.Models.Cars
+{
+    internal class Cars
```
Comment (ShriHumrudha) [inline CarApp.ConsoleApp/Models/Cars/Cars.cs:7]: The model holds just one entity so go with singular 'Car'

File: CarApp.ConsoleApp/Models/Cars/Cars.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    {
+        public int vinNumber { get; set; }
```
Comment (ShriHumrudha) [inline CarApp.ConsoleApp/Models/Cars/Cars.cs:9]: The properties should be in Camel Casing - VinNumber, with a capital V

File: CarApp.ConsoleApp/Models/Cars/Cars.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace Car.ConsoleApp.Models.Cars
```
Comment (ShriHumrudha) [inline CarApp.ConsoleApp/Models/Cars/Cars.cs:5]: The namespace should be - CarApp.ConsoleApp.Models.Cars

File: CarApp.ConsoleApp/Broker/Storages/StorageBroker.Cars.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+// ------------------------------------------------
+
+using System.Collections.Generic;
+using Car.ConsoleApp.Models.Cars;
+
+namespace Car.ConsoleApp.Broker.Storages
+{
+    internal partial class StorageBroker : IStorageBroker
+    {
+        List<Cars> _Cars = new List<Cars>();
```
Comment (ShriHumrudha) [inline CarApp.ConsoleApp/Broker/Storages/StorageBroker.Cars.cs:12]: We can skip the _, just Cars

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by ShriHumrudha** [comment]
Your PR title should be with 'BROKER:' prefix,

File: CarApp.ConsoleApp/Models/Car/Car.cs [hunk 5]
```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace CarApp.ConsoleApp.Models.Car
```
Comment (ShriHumrudha) [inline CarApp.ConsoleApp/Models/Car/Car.cs:5]: The folder name has to be plural - Cars

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: CarApp.ConsoleApp/Models/Car/Car.cs (hunk 5 above)
Comment (MananSThakkar) [reply CarApp.ConsoleApp/Models/Car/Car.cs]: Update the Model folder to "Cars"

File: CarApp.ConsoleApp/Broker/Storages/IStorageBroker.Car.cs [hunk 6]
```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using CarApp.ConsoleApp.Models.Cars;
+
+namespace CarApp.ConsoleApp.Broker.Storages
```
Comment (ShriHumrudha) [inline CarApp.ConsoleApp/Broker/Storages/IStorageBroker.Car.cs:7]: Brokers should be plural

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: CarApp.ConsoleApp/Broker/Storages/IStorageBroker.Car.cs (hunk 6 above)
Comment (MananSThakkar) [reply CarApp.ConsoleApp/Broker/Storages/IStorageBroker.Car.cs]: Updated folder name

File: CarApp.ConsoleApp/Brokers/Storages/IStorageBroker.Car.cs [hunk 7]
```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using CarApp.ConsoleApp.Models.Cars;
+
+namespace CarApp.ConsoleApp.Broker.Storages
```
Comment (ShriHumrudha) [inline CarApp.ConsoleApp/Brokers/Storages/IStorageBroker.Car.cs]: Even the namespace has to be updated - 'Brokers'

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**